server/service/src/stock_line/historical_stock.rs

```diff
@@ -63,7 +63,7 @@ pub fn get_historical_stock_lines_available_quantity(
         min_available_and_pack_size
             .entry(stock_line_id)
             .and_modify(|m| {
-                m.total = m.total - quantity;
+                m.total -= quantity;
```

Comment:
Is this a Clippy change? I prefer the old way :(

Reply:
I prefer this change o;

Reply:
I think this one was purely subjective! I'm not attached to though, just passing through.

I've actually turned clippy off since about halfway through doing this fix - it was just waaaay too slow on our codebase. Cargo check takes a few seconds, but clippy was like 30s+. It was so painful!